geomesa-ui/src/main/ui/src/app/card/card.js

```diff
@@ -41,4 +41,16 @@ angular.module('geomesa.card', [])
                 }, true);
             }
         };
-    }]);
+    }])
+
+    .directive('drag', function(){
+        return {
+
```

Comment:
Updated the restriction with 430737c commit and agree with you since the function is used to decorate an existing card element.